libraries/AP_Mount/AP_Mount_Gremsy.cpp

```diff
@@ -11,6 +11,11 @@ extern const AP_HAL::HAL& hal;
 #define AP_MOUNT_GREMSY_SEARCH_MS  60000    // search for gimbal for 1 minute after startup
 #define AP_MOUNT_GREMSY_ATTITUDE_INTERVAL_US    20000  // send ATTITUDE and AUTOPILOT_STATE_FOR_GIMBAL_DEVICE at 50hz
 
+void AP_Mount_Gremsy::init()
```

Comment:
you can actually just delete AP_Mount_Gremsy::init() and you'll find that the AP_Mount_Backend default implementation of init gets called.  This saves flash so it's worth it.  We're running low on flash (aka code space) on some of our boards/vehicles so it's worth removing.

Same goes for some of the others like Scripting..

Reply:
@rmackay9 , Done